sycl/source/detail/jit_compiler.cpp

```diff
@@ -74,6 +77,31 @@ jit_compiler::jit_compiler() {
       return false;
     }
 
+    this->CompileSYCLHandle = reinterpret_cast<CompileSYCLFuncT>(
+        sycl::detail::ur::getOsLibraryFuncAddress(LibraryPtr, "compileSYCL"));
+    if (!this->CompileSYCLHandle) {
+      printPerformanceWarning(
+          "Cannot resolve JIT library function entry point");
```

Comment:
This sounds more serious than a mere performance warning :)

Reply:
`printPerformanceWarning` is the generic error message helper in sycl-jit, but yes, I agree it's a bit of a misnomer when used here (and while attempting to set-up the other entrypoints before).